APMrover2/radio.cpp

```diff
@@ -61,12 +61,6 @@ void Rover::rudder_arm_disarm_check()
         return;
     }
 
-    // check if arming/disarming allowed from this mode
-    if (!control_mode->allows_arming_from_transmitter()) {
-        rudder_arm_timer = 0;
```

Comment:
this case is lost

Reply:
@khancyr so should I leave this check here as it is, and not remove it?

Reply:
@rishabsingh3003 zeroing the timer counter *might* not be important, but we definitely shouldn't be changing it accidentally.

This check should stay.  In time it will probably move to RC_Channels - it's been discussed before :-)